tests/get_kv_table_nodeos_tests.cpp

```diff
@@ -796,6 +1040,55 @@ BOOST_FIXTURE_TEST_CASE( get_kv_table_nodeos_test, TESTER ) try {
    chk_result(0, 4);
    chk_result(1, 5);
 
+   // test next_key
+   p.reverse = false;
+   p.index_name = "ff"_n;
+   p.index_value = "";
+   p.encode_type = "dec";
+
+   p.lower_bound = "-0.02";
+   p.upper_bound = "-0.01";
+   result = plugin.read_only::get_kv_table_rows(p);
+   BOOST_REQUIRE_EQUAL(0u, result.rows.size());
+   BOOST_REQUIRE_EQUAL(result.more, false);
+
+   p.upper_bound = "";
+   p.limit = 2;
+   result = plugin.read_only::get_kv_table_rows(p);
+   BOOST_REQUIRE_EQUAL(2u, result.rows.size());
+   BOOST_REQUIRE_EQUAL(result.more, true);
+   BOOST_REQUIRE_EQUAL(result.next_key.substr(0, 4) == "2.02", true);
```

Comment:
change all these to be .find("2.02") != std::npos, to avoid this failing with an exception if for some reason next_key is less than 4 characters long

Reply:
Fixed.